tox.ini

```diff
@@ -200,8 +200,8 @@ allowlist_externals =
     bash
 commands =
     bash .evergreen/setup-libmongocrypt.sh
-    bash {env:DRIVERS_TOOLS}/.evergreen/csfle/setup_secrets.sh
-    bash {env:DRIVERS_TOOLS}/.evergreen/csfle/start_servers.sh
+    bash {env:DRIVERS_TOOLS}/.evergreen/csfle/setup-secrets.sh
```

Comment:
Is this naming standard? Curious. I prefer - to _ myself, not aesthetically, just because it's easier to type!